xcell/mappers/mapper_DESY1wl.py

```diff
@@ -33,7 +33,13 @@ def __init__(self, config):
         self.nz = None
         # load cat
         self.cat_data = self._load_catalog()
+        # get items for calibration
+        self._get_Rs()
```

Comment:
I tend to prefer that `get_something` returns `something`; i.e. I would change that to `self.Rs = self._get_Rs()`